### PR TITLE
Make blockquotes more pretty and make citation more clear

### PR DESCRIPTION
**Pull request by Idadelveloper**
@jwflory this PR handles the styling for issue [#14](https://github.com/unicef/inventory/issues/14) in the [inventory](https://github.com/unicef/inventory) repo.

PS: My vs code formatter kinda made changes throughout the css file. The changes I really made are from line 594.

### CODE REVIEW

File: layouts/partials/head.html [hunk 1]
```diff
@@ -17,7 +17,11 @@
   <link rel="icon" href="{{ `images/favicon.png` | absURL }}" type="image/x-icon">
 
   {{ "<!-- fonts -->" | safeHTML }}
-  <link href="https://fonts.googleapis.com/css?family={{replace .Site.Params.font_family ' ' '+' | title}}:300,400,700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family={{replace .Site.Params.font_family_a ' ' '+' | title}}+{{replace .Site.Params.font_family_b ' ' '+' | title}}:wght@300;400;700&display=swap" rel="stylesheet">
```
Comment (jwflory) [inline layouts/partials/head.html:21]: Hmm, I am curious. Why was this change necessary? Not sure I follow your path of thinking.

File: layouts/partials/head.html [hunk 2]
```diff
@@ -17,7 +17,11 @@
   <link rel="icon" href="{{ `images/favicon.png` | absURL }}" type="image/x-icon">
 
   {{ "<!-- fonts -->" | safeHTML }}
-  <link href="https://fonts.googleapis.com/css?family={{replace .Site.Params.font_family ' ' '+' | title}}:300,400,700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
```
Comment (jwflory) [inline layouts/partials/head.html:20]: Is this a performance gain for loading font caches or something earlier? Sorry if it is an obvious question, I am curious. 🙊

File: assets/css/style.css [hunk 3]
```diff
@@ -18,869 +18,940 @@ h3,
...
 }
 
-blockquote p {
-  margin-bottom: 0 !important;
+
+/* Citation text */
+
+.user-quote blockquote p {
+    text-align: center;
+    /*padding-bottom: 3rem;*/
```
Comment (jwflory) [inline assets/css/style.css:663]: This looks much better than the original blockquote! 👍🏻 Also nice and tidy, easy to read.

**Review by jwflory — CHANGES_REQUESTED** [review]
Hi @Idadelveloper, this looks great and I am almost ready to merge this! The only thing that is a red flag to me for this Pull Request is the way fonts are loaded into the theme from the config. It is possible there is a bug here you caught working on this, but if it is a bug, we should look at fixing this another way that does not depend on the config file values.

Does this make sense? Let me know if you have doubts. Thanks again for working on this! 🙌🏻

File: layouts/partials/head.html (hunk 1 above)
Comment (Idadelveloper) [reply layouts/partials/head.html:21]: Remember 

> Hi @Idadelveloper, this looks great and I am almost ready to merge this! The only thing that is a red flag to me for this Pull Request is the way fonts are loaded into the theme from the config. It is possible there is a bug here you caught working on this, but if it is a bug, we should look at fixing this another way that does not depend on the config file values.
> 
> Does this make sense? Let me know if you have doubts. Thanks again for working on this! 🙌🏻

@jwflory  there's definitely a bug. Will get rid of this change together with the changes in the config file.

File: layouts/partials/head.html (hunk 2 above)
Comment (Idadelveloper) [reply layouts/partials/head.html]: @jwflory really has nothing much to do with performance. So when "Open+Sans" is used it is not being parsed correctly therefore an error shows in the console.
inventory error

I just removed the changes I made in this file and the config file. We could figure out a better way to tackle this :)

**Review by jwflory — APPROVED** [review]
Excellent. Looks good to me! Going to merge this and update the git submodule in the upstream inventory repo shortly.

Merging! 🎬

File: layouts/partials/head.html (hunk 2 above)
Comment (jwflory) [reply layouts/partials/head.html]: @Idadelveloper Good catch here. I didn't notice this bug until now. You could open an issue on [unicef/inventory](https://github.com/unicef/inventory) so we could talk through how best to fix that one.
